.github/workflows/ci.yml

```diff
@@ -9,12 +9,24 @@ on:
     types: [published]
 
 jobs:
+  validate-pr:
+    if: ${{ github.event_name == 'pull_request' }}
+    uses: mekomsolutions/shared-github-workflow/.github/workflows/maven-build-test.yml@main
+    with:
+      java-version: "8"
+      maven-phase: "install"
+      maven-args: "-P validator"
+    secrets: # We don't need secrets here. we should make them not required.
+      NEXUS_USERNAME: ${{ secrets.MEKOM_NEXUS_USERNAME }}
+      NEXUS_PASSWORD: ${{ secrets.MEKOM_NEXUS_PASSWORD }}
+
   validate:
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
     uses: mekomsolutions/shared-github-workflow/.github/workflows/maven-build-test.yml@main
     with:
       java-version: "8"
       maven-phase: "install"
-      maven-args: "-P prod" # Remove -Pvalidator to temporarily bypass validation
```

Comment:
Could you still keep the comment which indicates that the validation is disabled?